src/bench2.c

```diff
@@ -19,7 +19,7 @@
 #define BENCH_OUTLEN 16
 #define BENCH_INLEN 16
 
-static double min(const double *samples, size_t count)
+static double min_(const double *samples, size_t count)
```

Comment:
Could you please rename the function to `pick_min` instead? Other than that I'm fine with the changes. Thanks!